### PR TITLE
Rename status event level names

### PR DESCRIPTION
**Pull request by justinvdm**
At the moment we use `good`, `minor` and `major`. The plan is to change these to `ok`, `degraded` and `down`, since those are better descriptions for what the status events mean.

### CODE REVIEW

**Comment by justinvdm** [comment]
Ready for review.

**Comment by jerith** [comment]
I'd be happier if these were named constants, so it's harder to accidentally misspell them or something.

**Comment by hodgestar** [comment]
@jerith Doesn't message validation already check that?

**Comment by jerith** [comment]
Ah, I didn't see that. Thanks. :-)

I still think it makes more sense to use something more "enumeration-like" than arbitrary strings given that there is a fixed set of valid values, but I'm also okay with strings+validation.

**Comment by justinvdm** [comment]
@hodgestar Hmm, forgot that we do validation, good point.

Changed to using class-level variables, but happy to revert the change.

**Comment by hodgestar** [comment]
I'm okay with keeping the constants. They're sometimes a little annoying in Python because one ends up importing weird modules and classes just to access one string, but in Vumi itself that's not really an issue.

**Comment by hodgestar** [comment]
And we have validation too, so woot.

File: vumi/transports/tests/test_base.py [hunk 1]
```diff
@@ -123,12 +123,12 @@ def test_publish_status(self):
         })
 
         msg = yield transport.publish_status(
-            status='major',
+            status='down',
```
Comment (hodgestar) [inline vumi/transports/tests/test_base.py]: Do we want to use `TransportStatus.STATUS_DOWN`, etc in `test_base.py` too?

**Comment by hodgestar** [comment]
Other than the final tiny comment about a missed opportunity to used `TransportStatus.STATUS_X`, looks good to me.

**Comment by justinvdm** [comment]
Decided to revert the class-level variables approach, agree with [this comment](https://github.com/praekelt/vumi/pull/992#issuecomment-149237309).

Ready for re-review.

**Comment by hodgestar** [comment]
:+1:

**Pull request merged**